dev/TEAL_opcodes.md

```diff
@@ -618,6 +618,27 @@ See `bnz` for details on how branches work. `b` always jumps to the offset.
 - immediately fail unless A is a non-zero number
 - Availability: v3
 
+## bury n
+
+- Opcode: 0x45 {uint8 depth}
+- Stack: ..., A &rarr; ...
+- Replace the Nth value from the top of the stack. bury 0 fails.
```

Comment:
Replace it by which value? I'm guessing by `A` so I would write:

- Replace the Nth value from the top of the stack by A (The top value of the stack). bury 0 fails.

What happens if the stack has depth less than N+1?
Does N=1 means the value before the top of the stack?

Reply:
I'll say "Replace the Nth value from the top of the stack with A."

It panics if stack < N +1.  All of these sorts of opcodes do. It doesn't seem terribly useful to keep saying it. What else could they possibly do?  (bury, dig, cover, uncover, frame_bury, frame_dig, popn, )